base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -142,8 +142,13 @@ impl ConsensusConstants {
 
     pub fn coinbase_weight(&self) -> u64 {
         // TODO: We do not know what script, features etc a coinbase has - this should be max coinbase size?
+        let output_features_estimate = OutputFeatures {
+            // This is a best guess to not underestimate metadata size
+            recovery_byte: u8::MAX,
```

Comment:
It makes a difference for consensus encoding

Reply:
but it's encoded as a varint? cc @sdbondi

Reply:
and for a byte value it shouldn't be a varint?

Reply:
I doubt the recovery byte won't need to grow past 1 byte, so I suggest consensus encoding just writes the single byte (rather than varint) which means it will always be a fixed size regardless of the value.

Reply:
Changed recovery byte encoding to always be encoded to 1 byte